workflow/executor/os-specific/command.go

```diff
@@ -0,0 +1,25 @@
+package os_specific
+
+import (
+	"io"
+	"os"
+	"os/exec"
+
+	log "github.com/sirupsen/logrus"
+	"golang.org/x/term"
+)
+
+func simpleStart(cmd *exec.Cmd) (func(), error) {
+	if err := cmd.Start(); err != nil {
+		return nil, err
+	}
+
+	return func() {}, nil
+}
+
+func isTerminal(stdin io.Reader) bool {
+	f, ok := stdin.(*os.File)
+
+	log.WithField("argo", true).WithField("ok", ok).WithField("isterm", term.IsTerminal(int(f.Fd()))).Warn("checking term status of stdin")
```

Comment:
Can we change this to `Info`?

Reply:
This is temporary, I'm trying to debug my e2e test that succeeds locally  but fails in ci.

Reply:
logging removed and bug fixed